app/assets/stylesheets/styles/post-edit.scss

```diff
@@ -35,7 +35,7 @@ label {
   margin-left: 20px;
 }
 
-#middle-container {
+#edit-middle-container {
```

Comment:
Use a class

Reply:
This was given the class editMiddleContainer and the id was removed